nzedb/processing/tv/TV.php

```diff
@@ -191,37 +203,45 @@ public function setVideoNotFound($status, $Id)
 	 */
 	public function add($showArr = array())
```

Comment:
array $variable is a standard way to force calling code to pass an array. If it doesn't an error is generated, making tracking down the problem much easier.
= [] is the standard style we use for defaulting to empty array if nothing is provided.